mail/gnupg2/Portfile

```diff
@@ -81,10 +80,16 @@ variant pinentry_mac conflicts pinentry description {Handle user input via pinen
     configure.args-append   --with-pinentry-pgm=${applications_dir}/pinentry-mac.app/Contents/MacOS/pinentry-mac
 }
 
+variant openldap description {Build with openldap.} {
+    depends_lib-append      port:openldap
```

Comment:
Please get rid of the trailing dot in the variant description.

This is incomplete. If openldap is installed, it would probably be linked opportunistcally. You need to additionally pass flags that disable openldap and only enable it if +openldap is selected.

Not tested, but something like this is needed:

```
configure.args-append --disable-openldap
variant openldap description {Support openldap} {
    depends_lib-append	    port:openldap
    configure.args-replace  --disable-openldap --enable-openldap
}
```

Reply:
I would prefer for it not to be the default variant, but if he's going to insist, fine, leave it the default, but let people build without it if they want to.